src/blocks/_fhi-datepicker.block.scss

```diff
@@ -5,6 +5,16 @@
         display: block;
     }
 
+    &__description {
+        color: $fhi-grey-3;
+        display: block;
+        margin-bottom: $fhi-space-1;
+
+        .fhi-datepicker__label + & {
```

Comment:
Will we ever need more than one `fhi-datepicker__description`-element?

Reply:
No, but I've taken into account that the description might not necessarily be following a `fhi-datepicker__label`. The label might be missing

Reply:
Aha! I see.